src/audio/mixer.c

```diff
@@ -505,6 +505,9 @@ static void mixer_exec(int32_t *out, int num_samples) {
 				// actually present in the waveform.
 				if (wpos+wlen > len)
 					wlen = len-wpos;
+				// FIXME: due to a limit in the RSP ucode, we need to overread
```

Comment:
What needs fixing here? I presumed all converters would zero-pad the samples.

Reply:
I think the idea is: fixing the ucode so that the RSP doesn't have that limit anymore, assuming it can be done without waiving too much performance.